tools/cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -1522,4 +1518,37 @@
   {
     "id": "sorts a list alphabetically by order of [BASE_PATH | API_NAME], API_PATH, then API_VERB; only applicable within the limit/skip returned entity block",
     "translation": "sorts a list alphabetically by order of [BASE_PATH | API_NAME], API_PATH, then API_VERB; only applicable within the limit/skip returned entity block"
-  }]
+  },
+  {
+    "id": "prints bash command completion script to stdout",
+      "translation": "prints bash command completion script to stdout"
+  },
+  {
+    "id": "save action code to file that corresponds to the action name",
+    "translation": "file to save action code to"
```

Comment:
seems like for every `id === translation` (which is being fixed on its own IIRC). Is it different here on purpose?

Reply:
Good catch, typo.